### PR TITLE
comments should be converted to html before mailing 

### LINKED ISSUE #274: comments should be converted to html before mailing
junction comment system supports markdown. when somebody leaves a comment, it should be converted to html & then it should be send as html in emails.

### PR DESCRIPTION
**Pull request by ChillarAnand**
closes #274 

screenshot:
em

### CODE REVIEW

**Comment by kracekumar** [comment]
@ChillarAnand I think `pandoc` must be installed, can you add that to `travis.yml` ?

**Comment by kracekumar** [comment]
Also add a note in `readme` about `pandoc installation link`.

**Comment by ChillarAnand** [comment]
@kracekumar done. travis not sending mail notifications for failure?

File: README.md [hunk 1]
```diff
@@ -12,6 +12,13 @@ Junction is a software to manage proposals, reviews, schedule, feedback during c
 Setup
 =====
 
+Junction uses `pandoc` for converting markdown to html.
+To install run this command in Your terminal.
+
+```
+sudo apt-get install pandoc
```
Comment (kracekumar) [inline README.md]: Can you add link to pandoc installation `http://pandoc.org/installing.html` ?

Comment (ChillarAnand) [inline README.md]: @kracekumar updated

**Comment by kracekumar** [comment]
@ChillarAnand How about using pure python implementation ? https://github.com/trentm/python-markdown2. Sorry, about point it now. By using `pandoc` we're adding one more system dependency.

**Comment by ChillarAnand** [comment]
@kracekumar  no issues :) 
markdown2 :+1:

**Comment by kracekumar** [comment]
@vigneshsarma Deploy ?

**Pull request merged**